tzconvert.py

```diff
@@ -54,9 +65,9 @@ def adjust_datetime(naive_date, city_name, debug=False):
         naive_datetime = current_datetime
         adjusted_datetime = adjust_datetime(naive_datetime, city)
 
-        if not adjusted_datetime:
-            print("Could not fetch adjusted date time")
-            continue
+    latitude, longitude = 22.5650515, 88.391722
+    nearest_city_name = tzhelpers.get_nearest_city(latitude, longitude)
+    approximate_datetime = approx_datetime(current_datetime, latitude, longitude)
```

Comment:
These should be moved further down after the demo print statements of the first module demo.

Reply:
Yes, will make the changes and push.